text/work_packages_and_reports.tex

```diff
@@ -155,7 +155,7 @@ \subsection{Computation of Work Results}\label{sec:computeworkresult}
     (\mathbb{P}, \N_\mathsf{C}) &\to \mathbb{W} \\
     (\wpX, c) &\mapsto \begin{cases}
         \error &\when \mathbf{o} \not\in \Y \\
-        \tup{\is{a}{\wpX_\wp¬authorizer}, \mathbf{o}, \is{x}{\wpX_\wp¬context}, s, \mathbf{l}, \mathbf{r}} \!\!\!\!\!&\otherwise
+        \tup{s, \is{x}{\wpX_\wp¬context}, c, \is{a}{\wpX_\wp¬authorizer}, \mathbf{o}, \mathbf{l}, \mathbf{r}} \!\!\!\!\!&\otherwise
```

Comment:
Not a big deal, but I'd prefer if this tuple fields are listed with the same order as the work report ( $\mathbb{W}$ ) ones